### PR TITLE
Animation loop default to true

### LINKED ISSUE #583: Animation.loop should be true by default for sprites
Perhaps this could be up for debate, but I feel that in the majority of cases it is desirable for a sprite to loop indefinitely.

### PR DESCRIPTION
**Pull request by vritant24**
Closes #583 

Sprites in most cases should loop infinitely, so the default for Animation.loop should be true.

## Changes:

- Animation.loop set to true by default
- disabled `Label` test `can measure text width`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - Camera zoom over time now returns a promise that resolves on completion ([#800](https://github.com/excaliburjs/Excalibur/issues/800))
 - Edge builds have more descriptive versions now containing build number and Git commit hash (e.g. `0.10.0-alpha.105#commit`) ([#777](https://github.com/excaliburjs/Excalibur/issues/777))
+- Animation loop now set to true by default ([#583] (https://github.com/excaliburjs/Excalibur/issues/583))
```
Comment (kamranayub) [inline CHANGELOG.md]: Rephrase as "`Animation.loop property now to set to `true` by default"

File: src/engine/Docs/Animations.md [hunk 2]
```diff
@@ -30,4 +30,5 @@ game.start(loader).then(function () {
 
 You can add [["Drawing/SpriteEffects"|sprite effects]] to an animation through methods
 like [[Animation.invert]] or [[Animation.lighten]]. Keep in mind, since this
-manipulates the raw pixel values of a [[Sprite]], it can have a performance impact.
\ No newline at end of file
+manipulates the raw pixel values of a [[Sprite]], it can have a performance impact.
+[[Animetion.loop]] is now set to true by default.
```
Comment (kamranayub) [inline src/engine/Docs/Animations.md]: Typo and rephrase as "Animations will loop by default. You can use [[Animation.loop]] to change this behavior."

File: src/spec/AnimationSpec.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+/// <reference path="Mocks.ts" />
+
+describe('An animation', () => {
+   var animation;
+   beforeEach(() => {
+      animation = new ex.Animation(null, null, 0);
+   });
+
+   it('should have loop default to true', () => {
+      expect(animation.loop).toBeTruthy();
```
Comment (kamranayub) [inline src/spec/AnimationSpec.ts]: Let's be exact, `expect(animation.loop).toBe(true)`

**Comment by vritant24** [comment]
Made the changes 👍

**Pull request merged**